### PR TITLE
Rewording the default arguments warnings

### PR DESCRIPTION
**Pull request by kelvinst**
Refs #10095 

I took the liberty of also rewording the other warnings about arguments with default values, so they all look alike.

### CODE REVIEW

File: lib/elixir/src/elixir_locals.erl [hunk 1]
```diff
@@ -121,13 +121,13 @@ format_error({function_conflict, {Receiver, {Name, Arity}}}) ->
     [elixir_aliases:inspect(Receiver), Name, Arity]);
 
 format_error({unused_args, {Name, Arity}}) ->
-  io_lib:format("default arguments in ~ts/~B are never used", [Name, Arity]);
+  io_lib:format("default values for the arguments in ~ts/~B are never used", [Name, Arity]);
 
 format_error({unused_args, {Name, Arity}, 1}) ->
-  io_lib:format("the first default argument in ~ts/~B is never used", [Name, Arity]);
+  io_lib:format("the default value for the first argument in ~ts/~B is never used", [Name, Arity]);
```
Comment (josevalim) [inline lib/elixir/src/elixir_locals.erl:127]:
```suggestion
  io_lib:format("the default value for the first optional argument in ~ts/~B is never used", [Name, Arity]);
```

?

File: lib/elixir/src/elixir_locals.erl [hunk 2]
```diff
@@ -121,13 +121,13 @@ format_error({function_conflict, {Receiver, {Name, Arity}}}) ->
...
 
 format_error({unused_args, {Name, Arity}, Count}) ->
-  io_lib:format("the first ~B default arguments in ~ts/~B are never used", [Count, Name, Arity]);
+  io_lib:format("the default values for the first ~B arguments in ~ts/~B are never used", [Count, Name, Arity]);
```
Comment (josevalim) [inline lib/elixir/src/elixir_locals.erl:130]:
```suggestion
  io_lib:format("the default values for the first ~B optional arguments in ~ts/~B are never used", [Count, Name, Arity]);
```

?

File: lib/elixir/src/elixir_locals.erl (hunk 1 above)
Comment (kelvinst) [reply lib/elixir/src/elixir_locals.erl]: Done

File: lib/elixir/src/elixir_locals.erl (hunk 2 above)
Comment (kelvinst) [reply lib/elixir/src/elixir_locals.erl]: Done

**Comment by josevalim** [comment]
@kelvinst can you please run the formatter? Then this should be good to go!

**Comment by kelvinst** [comment]
> @kelvinst can you please run the formatter? Then this should be good to go!

Done!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**